src/dashboard/components/files/FileActions.js

```diff
@@ -36,6 +36,8 @@ export default function FileActions({ missing, id, filePath, openFile }) {
     switch (eventKey) {
       case 'open':
         openFile(filePath, id)
+        shell.showItemInFolder(filePath)
```

Comment:
I think this line was added in error. It's not needed here, but the `break` below this line is definitely needed. Good catch

Reply:
Ja.  I thought it might be but wasn't sure.
EsLint pointed it out as a potential mistake, and I wanted to start by retaining the behaviour while removing the fall-through.

I've gone ahead and removed it.